src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -29,14 +27,15 @@ interface RoomModalProps {
 }
 
 export const RoomModal: React.FC<RoomModalProps> = ({ show, onHide, room }) => {
+  useFirestoreConnect("venues");
   const { user, profile } = useUser();
 
   const venue = useSelector(currentVenueSelector);
   const venues = useSelector(orderedVenuesSelector);
   const venueEvents = useSelector(
     (state) => state.firestore.ordered.venueEvents
   );
```

Comment:
[OOS] We should refactor this to use a selector function in `selectors.ts`

See https://github.com/sparkletown/internal-sparkle-issues/issues/115